### PR TITLE
Added XGG Token to Default

### PR DESCRIPTION
**Pull request by GoingGems**
Going Gems are Free to use for everyone that can claimed by send 0 ETH  to XGG Contract Adrress and can get maximum 5000 XGG. Main project is Game that can be purchasing by the token.

### CODE REVIEW

File: tokens/tokens-eth.json [hunk 1]
```diff
@@ -1,27 +1,29 @@
 [
 	{
-		"symbol": "$TEAK",
```
Comment (gamalielhere) [inline tokens/tokens-eth.json]: Did you mean to change the coin $TEAK?

**Comment by GoingGems** [comment]
oh I'm sorry,  we Mean "XGG"

Regards
XGG Team

**Comment by gamalielhere** [comment]
@GoingGems Sounds good. Please update the PR without updating other tokens.

**Comment by GoingGems** [comment]
what you mean of PR???

**Comment by GoingGems** [comment]
Just see on Green Block,  not Red Block,
On Apr 11, 2018 4:36 PM, "Going Gems" <going.gems1@gmail.com> wrote:
>
> what you mean of PR???
>
> On Apr 11, 2018 3:58 AM, "gamalielhere" <notifications@github.com> wrote:
>>
>> @GoingGems Sounds good. Please update the PR without updating other
tokens.

**Comment by gamalielhere** [comment]
@GoingGems I mean that the Pull request (PR) you submitted is changing another token. Your pull request should only be adding your token.

**Comment by gamalielhere** [comment]
moved to #633

**Pull request closed without merging**